tests/summarizer/test_openai_summarizer.py

```
@@ -0,0 +1,315 @@
+        # Mock summarizer
+        mock_summarizer = Mock()
+        mock_summarizer.generate_summary.return_value = (
+            "テストタイトル",
+            "テスト説明文",
+            ["トピック1"]
+        )
+        mock_summarizer.save_summary.return_value = output_file
+        mock_summarizer_class.return_value = mock_summarizer
+
+        # Summarize
+        title, description, topics, summary_file = summarize_transcript(
```

Ruff (0.14.5)

Unpacked variable `title` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)